app/views/scans/_ineligible_exception.html.erb

```diff
@@ -0,0 +1,5 @@
+<strong class="h3">Scan-to-PDF not available.</strong>
+
+<p>
+This pilot service is currently available only to faculty, grad students, postdocs, academic staff, fellows, and visiting scholars. Your feedback will help us refine and expand the service!
+</p>
```

Comment:
:+1: can we remove the i18n key so there is no confusion about where that text comes from?